messages/scratchOrgCreate.md

```diff
@@ -0,0 +1,7 @@
+# unsupportedSnapshotOrgCreateOptions
```

Comment:
All these error names need to follow this format: https://github.com/forcedotcom/sfdx-core/blob/v3/MIGRATING_V2-V3.md#error-names